docs/REST API/rest-api.md

```diff
@@ -3891,6 +3891,7 @@ Add a configuration profile to enforce custom settings on macOS and Windows host
 | ------------------------- | -------- | ---- | ------------------------------------------------------------------------------------------------------------- |
 | profile                   | file     | form | **Required.** The .mobileconfig (macOS) or XML (Windows) file containing the profile. |
 | team_id                   | string   | form | _Available in Fleet Premium_ The team ID for the profile. If specified, the profile is applied to only hosts that are assigned to the specified team. If not specified, the profile is applied to only to hosts that are not assigned to any team. |
+| labels                   | array     | form | An array of labels to filter hosts in a team (or no team) that should get a profile. |
```

Comment:
```suggestion
| label                   | array     | form | An array of labels to filter hosts in a team (or no team) that should get a profile. |
```

I think we want this to be singular `label` right? (looking at your example). 

Also, I think `label` should be type `string` instead? (I could be wrong). Looking at the example, it looks like a user specifies `label` in the same way they do `team_id`.

I think we should clarify that you can specify one or more and point the user to the example below.

Reply:
I'm wondering if would it be weird if parameter is named `label` (singular) because in the UI we can select multiple labels.
@roperzh What's the most common way to handle this case (a form with an upload field and option to select multiple values)?

Reply:
if you can send more than one, then generally the name is in plural form (`labels`)

Reply:
@noahtalerman that's the way to specify an array of things in form data, multiple elements with the same name (`name="labels"`)

maybe printing the form data like that is not the best way to explain the intended behavior? we thought it was a good idea initially.

Reply:
Oh interesting. 

> maybe printing the form data like that is not the best way to explain the intended behavior?

How else could we print it?